addons/api/mirage/factories/alias.js

```diff
@@ -0,0 +1,20 @@
+/**
+ * Copyright (c) HashiCorp, Inc.
+ * SPDX-License-Identifier: MPL-2.0
+ */
+
+import factory from '../generated/factories/alias';
+import permissions from '../helpers/permissions';
+import generateId from '../helpers/id';
+
+export default factory.extend({
+  id: () => generateId('alt_'),
```

Comment:
nonblocking question for learning: Is the `alt_` prefix specific to the target type similar to `ttcp_` or `tssh_` for TCP and SSH targets?

Reply:
Yes, it is a prefix for alias IDs that the backend uses.